README.md

```diff
@@ -0,0 +1,52 @@
+
+### “needs author input”:
+The editors are seeking input from web authors on the issue. For example, whether a requested change is useful or how best to expose information.
+
+### “needs follow-up”:
+The assignee, likely an editor, needs to investigate more deeply before we can decide if this “needs implementation” or to otherwise move forward. The editors have discussed the issue and do not need to discuss it further until we have the resulting follow-up from the assignee. This includes things like determining external spec dependencies, seeking input from other spec owners and/or WGs, confirming the understanding of the nature of the bug, and beginning to formulate a path to a solution.
+
+This doesn’t necessarily mean follow-up has “Started” or is “In Progress.”
+
+### “needs implementation”:
+The steps needed to resolve this issue are clearly understood and agreed upon. This likely means drafting and committing a spec change, possibly via a pull request. No further discussion is necessary at this time, though review of the change may still be appropriate. Should that change, this label should be removed.
+
+For a bug to be labeled with this, it needs to be understood well enough and in scope of the marked milestone. Otherwise, “needs follow-up” or punting milestone might be options.
+
+This label does not refer to user agent implementations.
+
+### “blocked”:
+Some external dependency or another GitHub issue identified in comment associated with this label’s addition is (or might be) blocking progress on this bug.
+
+### “feature request”:
+The issue is related to or requesting a new use case or capability not currently (explicitly) covered by the spec. Depending on the nature and impact of the request and the stage of the spec, it may be assigned to a future milestone.
+
+### “interoperability”:
+Resolution of this issue is particularly important for interoperability among user agents. This may include breaking API changes, issues related to media compatibility across user agents, or ambiguous parts of the spec that could lead to different incompatible interpretations. There may be known or probable differing interpretations in implementations of the associated portion of the spec. If the identified issue is not addressed, there is a high likelihood of meaningful interoperability problems. The fix for the issue would need to provide a clear direction to prevent differing interpretations by user agents.
+
+### “wontfix”, “invalid”, “duplicate”:
+Self-explanatory :)
+
+Issues with these labels should always be closed (unless they were re-opened at which time an editor should probably remove these labels if the re-opening is accepted).
+
+
+## Milestones
+
+### “V#”:
+A bug in version # of the spec.
+Currently, V1 is the first version of MSE (in CR in Q4 2015).
+
+### “V.Next”:
```

Comment:
`VNext` probably makes more sense since that will make it alphabetically after V# (i.e. in the Milestones dropdown).

Alternatively, use a little 'v' in all cases: v1, vNext